modules/lofreq/1.0/lofreq.smk

```diff
@@ -143,7 +162,8 @@ rule _lofreq_all:
             genome_build=CFG["runs"]["tumour_genome_build"],
             tumour_id=CFG["runs"]["tumour_sample_id"],
             normal_id=CFG["runs"]["normal_sample_id"],
-            pair_status=CFG["runs"]["pair_status"])
+            pair_status=CFG["runs"]["pair_status"],
+            tool="lofreq")
```

Comment:
I would be surprised if this worked for more than one sample. For Strelka I had to use this syntax (you should double-check):

tool = "lofreq" * len(CFG["runs"])

Reply:
yes, you are right. It works only for one sample.

Reply:
I don't think it's necessary to have it as a wildcard here anyway. It can be called as a wildcard in downstream tools without it being specified here.

Reply:
That's a really good point. I had done it this way in the liftover tool because it made sense to have it as a wildcard there. Here it could just be explicitly "strelka" throughout. That said, I suppose there's no harm in this as long as it is expanded out to be repeated the right number of times. The only (minor) benefit of doing it this way is the changes you make to the code will be identical in the other variant callers. Aside: if the config stores the tool name then it would be even more elegant to use it from the config instead of hard-coding it in the expand statement. That would make the code even more copy-paste-able for modifying Mutect, Strelka etc.

Reply:
Yes, I agree this wildcard is not explicitly needed here. I thoght more of a "placeholder" here, since we might later want to include the version of the module as well. For that purpose, I will move this to config, so it will be easily modifiable in the future.